storage/dataflux/doc.go

```diff
@@ -0,0 +1,27 @@
+// Copyright 2024 Google LLC
```

Comment:
Should we add a README to this directory or something indicating that it's under active development?

Reply:
Added a README.md  with a description, usage example and some benchmarks.